### ISSUE TITLE
Could not create SSL/TLS secure channel.

### ISSUE THREAD

**Issue by FuzzGuitar** [RickardPettersson/swish-for-handel-csharp#1]
Any idea of how to handle the following error:
`The request was aborted: Could not create SSL/TLS secure channel`

Thank you!

**Comment by RickardPettersson** [comment]
Its depending totally on your setup of project. if you download the code and run the console application and try what result you get? Works local on my machine Windows 10, Visual Studio 2019.

**Comment by FuzzGuitar** [comment]
Thanks for answering!
I got passed the ssl by following this guide:
`https://dontpaniclabs.com/blog/post/2017/01/11/creating-self-signed-ssl-certificate-developing-testing-iis/`

What happens now is that I can successfully make a paymentrequest,´. But instead of asking me to open swish on my phone, it goes straight to the: Betalningen slutförd!

I must have messed up somewhere. But iĺl get it working. Thank you!

The javascript-function:
`CheckStatus();`
Gets called directly when the page gets rendered passing the `data.status == 'PAID'` which it gets from:
`PaymentStatus`

**Comment by RickardPettersson** [comment]
Install the certification is a workaround becouse the code is maked to not be necessary to install the certificate. But you are not the first one doing that to get it to work so I need to investigate this.

What OS do you using and version, Visual Studio version?

In the test environment Swish saying it is PAYED without using the Swish app.

**Comment by FuzzGuitar** [comment]
OS: Win10
VS: 2017

Aha ok, so in the test env there is no way to actually try it out using the phone?

**Comment by RickardPettersson** [comment]
No, you need a agreement with a Swedish bank and do it in production environment to get app to use.

**Comment by bagoose** [comment]
I'm getting this error as well. It's on my localhost on Windows 10 with Visual Studio 2019. I have just downloaded it and run the console app.

**Comment by RickardPettersson** [comment]
I getting the test certificate to work with the code without a problem but then i got to loan a persons certicate for production and that not works so i am now trying to get my own agreements with my bank and testing this out.
